Zilon.Core/Zilon.Core/Persons/CitizenPerson.cs

```diff
@@ -37,22 +38,32 @@ public class CitizenPerson : IPerson
 
         public CitizenType CitizenType { get; }
 
+        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)